packages/types/src/story/page.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import type { Element, Pattern, ShapeElement } from '../element';
+import type { Animation } from './animation';
+
+export interface Group {
+  name: string;
+  isLocked: boolean;
+  isCollapsed?: boolean;
+}
+
+export type Groups = Record<string, Group>;
+
+export interface Page {
+  elements: Element[];
```

Comment:
Should this specify that we have an array that can consist of text / shape / image etc. element types instead of just the general `Element[]` array? 🤔

Reply:
`Element[]` should be fine, since ideally in the future someone would be able to build their own custom story editor with custom element types